src/functions/kern.js

```diff
@@ -39,7 +39,7 @@ defineFunction({
         }
         return {
             type: "kern",
-            dimension: args[0].value,
+            dimension: args[0].value.value,
```

Comment:
The types for `args` weren't enough to pick that this needed to change.

Reply:
As the first line in this function, could you do: `const size = assertNodeType(args[0], "size");` and use that instead of `args[0]`? That will capture any type errors in its usage very precisely.